privilege check on temp-dir (#14235)
pingcap/docs: hardware-and-software-requirements.md

> - If you only want to test and verify the features, follow [Quick Start Guide for TiDB](/quick-start-with-tidb.md) to deploy TiDB on a single machine.
> - The TiDB server uses the disk to store server logs, so there are no special requirements for the disk type and capacity in the test environment.
> - Starting from v6.3.0, to deploy TiFlash under the Linux AMD64 architecture, the CPU must support the AVX2 instruction set. Ensure that `cat /proc/cpuinfo | grep avx2` has output. To deploy TiFlash under the Linux ARM64 architecture, the CPU must support the ARMv8 instruction set architecture. Ensure that `cat /proc/cpuinfo | grep 'crc32' | grep 'asimd'` has output. By using the instruction set extensions, TiFlash's vectorization engine can deliver better performance.

> > - Starting from v6.5.0, `Fast Online DDL` (controlled by variable [`tidb_ddl_enable_fast_reorg`](/system-variables.md#tidb_ddl_enable_fast_reorg-new-in-v630)) is enabled by default to accelerate DDL operations, such as adding indexes. If DDL operations involving large objects exist in the system, it is highly recommended to prepare additional SSD disk space for TiDB (100 GB+ is recommended). For detailed configuration instructions, please refer to [Set temporary space for TiDB instances]()

Reviewer: 这部分中文没有

Author: 这部分和下面是重合的，我移到下面了， 这句删掉

Reviewer: ok

| Component | Disk space requirement | Healthy disk usage |
| :-- | :-- | :-- |

> | TiDB | <ul><li>At least 30 GB for the log disk</li> <li> Starting from v6.5.0, `Fast Online DDL` (controlled by variable [`tidb_ddl_enable_fast_reorg`](/system-variables.md#tidb_ddl_enable_fast_reorg-new-in-v630)) is enabled by default to accelerate DDL operations, such as adding indexes. If DDL operations involving large objects exist in the system, it is highly recommended to prepare additional SSD disk space for TiDB (100 GB or more). For detailed configuration instructions, see [Set a temporary space for a TiDB instance](/check-before-deployment.md#set-a-temporary-space-for-a-tidb-instance-recommended) </li></ul> | Lower than 90% |

Reviewer: ```suggestion
| TiDB | <ul><li>At least 30 GB for the log disk</li> <li> Starting from v6.5.0, `Fast Online DDL` (controlled by variable [`tidb_ddl_enable_fast_reorg`](/system-variables.md#tidb_ddl_enable_fast_reorg-new-in-v630)) is enabled by default to accelerate DDL operations, such as adding indexes. If DDL operations involving large objects exist in the system, it is highly recommended to prepare additional SSD disk space for TiDB (100 GB or more). For detailed configuration instructions, see [Set a temporary space for a TiDB instance](/check-before-deployment.md#set-temporary-spaces-for-tidb-instances-recommended) </li></ul> | Lower than 90% |
```

| Component | Disk space requirement | Healthy disk usage |
| :-- | :-- | :-- |

> | TiDB | <ul><li>At least 30 GB for the log disk</li> <li> Starting from v6.5.0, `Fast Online DDL` (controlled by variable [`tidb_ddl_enable_fast_reorg`](/system-variables.md#tidb_ddl_enable_fast_reorg-new-in-v630)) is enabled by default to accelerate DDL operations, such as adding indexes. If DDL operations involving large objects exist in the system, it is highly recommended to prepare additional SSD disk space for TiDB (100 GB or higher). For detailed configuration instructions, please refer to [Set temporary space for TiDB instances](/check-before-deployment.md#set-temporary-space-for-tidb-instances-recommended) </li></ul> | Lower than 90% |

Reviewer: ```suggestion
| TiDB | <ul><li>At least 30 GB for the log disk</li> <li> Starting from v6.5.0, `Fast Online DDL` (controlled by variable [`tidb_ddl_enable_fast_reorg`](/system-variables.md#tidb_ddl_enable_fast_reorg-new-in-v630)) is enabled by default to accelerate DDL operations, such as adding indexes. If DDL operations involving large objects exist in the system, it is highly recommended to prepare additional SSD disk space for TiDB (at least 100 GB). For detailed configuration instructions, see [Set a temporary space for a TiDB instance](/check-before-deployment.md#set-a-temporary-space-for-a-tidb-instance-recommended) </li></ul> | Lower than 90% |
```

Author: > (at least 100 GB)

100 GB and higher is recommended for most cases. However, space less than 100GB still do its job with less performance.

Author: "(at least 100 GB)" implies less space doesn't work.

Reviewer: changed to 100 GB or more
